libs/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,23 @@
+import { AnimeDto } from '../dtos/anime.dto';
+import { Anime } from '../models/anime';
+
+import { DateRangeMapper } from './dateRange.mapper';
+
+export namespace AnimeMapper {
+
+  /**
+   * @param dto Anime dto.
+   */
+  export function fromDto(dto: AnimeDto): Anime {
+    const anime: Anime = {
+      id: dto.id,
+      titleEnglish: dto.title_eng,
+      titleJapan: dto.title_jpn,
+      image: dto.image,
+      aired: DateRangeMapper.fromDto(dto.aired),
+      type: dto.type,
+      status: dto.status,
+    };
+    return anime;
```

Comment:
You don't really need to create variable here.
```suggestion
    return {
      id: dto.id,
      titleEnglish: dto.title_eng,
      titleJapan: dto.title_jpn,
      image: dto.image,
      aired: DateRangeMapper.fromDto(dto.aired),
      type: dto.type,
      status: dto.status,
    };
```

Reply:
And in other mappers too